compiler/src/dotty/tools/dotc/interactive/Completion.scala

```diff
@@ -503,10 +505,10 @@ object Completion:
       // There are four possible ways for an extension method to be applicable
 
       // 1. The extension method is visible under a simple name, by being defined or inherited or imported in a scope enclosing the reference.
-      val termCompleter = new Completer(Mode.Term, prefix, pos)
-      val extMethodsInScope = termCompleter.scopeCompletions.toList.flatMap:
-        case (name, denots) => denots.collect:
-          case d: SymDenotation if d.isTerm && d.termRef.symbol.is(Extension) => (d.termRef, name.asTermName)
+      val termCompleter = new Completer(Mode.Term, prefix, pos, _.startsWith(prefix))
```

Comment:
Sorry if the question doesn't make much sense, I'm not really familiar with this code, but does it mean that extension methods will be excluded from the fuzzy search? :slightly_smiling_face: 

In any case, great to see this improvement finally coming to Scala 3 completions! :+1:

Reply:
This is my oversight. It should also be fuzzy. Great catch 👍🏻